src/main/java/org/opentripplanner/updater/car_rental/CarRentalUpdater.java

```diff
@@ -158,14 +164,13 @@ private void applyStations(Graph graph) {
                 CarRentalStationVertex vertex = verticesByStation.get(station);
                 if (vertex == null) {
                     makeVertex(graph, station);
-                } else if (station.x != vertex.getX() || station.y != vertex.getY()) {
+                } else if (Math.abs(station.x - vertex.getX()) > 0.00005 || Math.abs(station.y - vertex.getY()) > 0.00005) {
                     LOG.info("{} has changed, re-graphing", station);
```

Comment:
Can this comment be more precise and say that the station's position has changed?

Reply:
Will do